packages/node_modules/@webex/internal-plugin-mercury/src/mercury.js

```diff
@@ -179,15 +178,10 @@ const Mercury = WebexPlugin.extend({
           logger: this.logger
         };
 
-        // if a proxy is being used by the request module we will need to use it here as well to open a websocket
-        const proxyAddress = process.env.HTTPS_PROXY || process.env.HTTP_PROXY || null;
-
-        if (proxyAddress) {
+        // if a proxy agent has been configured we will add it here to successfully open a websocket
+        if (this.webex.config.defaultRequestOptions && this.webex.config.defaultRequestOptions.proxyAgent) {
           this.logger.info('mercury: setup proxy agent');
-
-          const proxyOption = url.parse(proxyAddress);
-
-          options.agent = new HttpsProxyAgent(proxyOption);
+          options.agent = this.webex.config.defaultRequestOptions.proxyAgent;
```

Comment:
May be better to create a separate defaultRequestOptions. e.g. defaultMercuryRequestOptions.